modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -173,6 +174,18 @@ We're removing memoization from focus ring. The `memoize` argument passed to `fo
 longer valid and we've removed the exported `memoizedFocusRing`. There is no codemod for this
 change. We couldn't find any example of `memoize` or `memoizedFocusRing` being used.
 
+### useThemedRing
+
+We've promoted `useThemedRing` from our Labs package to our Main package. You can use this utility
+to theme focus rings.
+
+🤖 The codemod will update the import for this utility.
+
+### useThemeRTL
+
+We've [soft-deprecated](#soft-deprecation) `useThemeRTL` from our Labs package. Although you may still use this utility, we encourage consumers to use
```

Comment:
Not sure if you want to update this now or later but, we will be switching the wording for "soft-dep" and "hard-dep" to "deprecate" and "removal"

Reply:
I'll keep as is and once this gets merge we can update all instances in the v9 upgrade guide? Unless the upgrade guide gets merged first

Reply:
Sounds good to me. This won't be done until next sprint.